### ISSUE TITLE
Log collection nil pointer

### ISSUE THREAD

**Issue by dwbrown2** [kubecost/cost-analyzer-helm-chart#452]
A recent release introduced the `reporting` block in Kubecost values.yaml. When `--reuse-values` is used and this field doesn't exist, the following error is created.

```
UPGRADE FAILED
Error: render error in "cost-analyzer/templates/cost-analyzer-cluster-role-binding-template.yaml": template: cost-analyzer/templates/cost-analyzer-cluster-role-binding-template.yaml:1:36: executing "cost-analyzer/templates/cost-analyzer-cluster-role-binding-template.yaml" at <.Values.reporting.logCollection>: nil pointer evaluating interface {}.logCollection
Error: UPGRADE FAILED: render error in "cost-analyzer/templates/cost-analyzer-cluster-role-binding-template.yaml": template: cost-analyzer/templates/cost-analyzer-cluster-role-binding-template.yaml:1:36: executing "cost-analyzer/templates/cost-analyzer-cluster-role-binding-template.yaml" at <.Values.reporting.logCollection>: nil pointer evaluating interface {}.logCollection
```

An immediate workaround is to pass this block in values.

```
reporting:
  logCollection: true 
  productAnalytics: true 
  errorReporting: true
```

**Comment by dwbrown2** [comment]
https://github.com/kubecost/cost-analyzer-helm-chart/pull/453

**Comment by dwbrown2** [comment]
Fixed in v1.58.1
